gapir/replay_service/service.proto

```diff
@@ -52,12 +52,17 @@ message Replay {
   string dependent_id = 2;
 }
 
+message ProfilerReady {
```

Comment:
This is the message sent when a WAIT is encountered, correct? Let's un-tie it from the fact that it's used for profiling and simply call it "FenceReady".